src/components/drawing/index.js

```diff
@@ -579,16 +579,17 @@ drawing.makeTester = function(gd) {
         tester.node()._cache = {};
     }
 
-    gd._tester = tester;
-    gd._testref = testref;
+    drawing.tester = gd._tester = tester;
+    drawing.testref = gd._testref = testref;
```

Comment:
`gd._testref` currently isn't used anywhere so we might as well get rid of it before someone *does* use it.

`gd._tester` *is* used in a few places, but wouldn't be hard to 🔪  and replace with `drawing.tester`.